Adding OSS announcement (#35)
mulesoft-labs/data-weave-rfc: README.md

# Data Weave Language

> The DataWeave Language is in the process of being Open Source. The [anoucement](https://blogs.mulesoft.com/news/dataweave/). The process has just began and it is going to take some time. In the meantime we want to start engaging with our community to understand how DataWeave wants to be used or integrated. Feel free to open a ticket or join our [community slack](https://join.slack.com/t/dataweavelanguage/shared_invite/zt-1f3xmq8n6-MVoUj7dDamxyu_Zyf62ERQ) 

Reviewer: ```suggestion
The DataWeave Language is in the process of being open-sourced. You can read our announcement [here](https://blogs.mulesoft.com/news/dataweave/). Our journey has just begun and it will take some time for the code to be available. In the meantime, we want to start engaging with our community to understand how DataWeave could be used and integrated. 
```

> Also if you want to be part of an Early Access Program:

Reviewer: ```suggestion
If you are interested on leveraging DataWeave:
```

>  1. Join our [slack](https://join.slack.com/t/dataweavelanguage/shared_invite/zt-1f3xmq8n6-MVoUj7dDamxyu_Zyf62ERQ)

Reviewer: ```suggestion
 1. Join our community [Slack](https://join.slack.com/t/dataweavelanguage/shared_invite/zt-1f3xmq8n6-MVoUj7dDamxyu_Zyf62ERQ)
```

>  2. Join the opensource slack channel

Reviewer: ```suggestion
 2. Join the `#opensource` channel
```

> For more news visit our [site](https://dataweave.mulesoft.com/) 

Reviewer: ```suggestion
For more news and all things DataWeave, visit our [site](https://dataweave.mulesoft.com/) 
```
